site/content/resources/_index.html

```diff
@@ -38,6 +38,41 @@ <h3>Here you will find external resources about Carvel, such as videos, workshop
            <p class="strong">Getting Started Guide by Leigh Capili & Tiffany Jernigan</p>
            <p><a href="https://tanzu.vmware.com/developer/guides/kubernetes/ytt-gs/">Getting Started with ytt</a>.</p>
        </div>
+   </div>    
+   <div class="col">
+       <div class="icon">
+           <img src="/img/logo.svg" />
+       </div>
+       <div class="content">
+           <p class="strong">Getting Started Guide by Leigh Capili & Tiffany Jernigan</p>
+           <p><a href="https://tanzu.vmware.com/developer/guides/kapp-controller-gs/">Getting Started with kapp-controller</a>.</p>
+       </div>
+   </div>
+   <div class="col">
+       <div class="icon">
+           <img src="/img/logo.svg" />
+       </div>
+       <div class="content">
+           <p class="strong">Getting Started Guide by Tiffany Jernigan</p>
+           <p><a href="https://tanzu.vmware.com/developer/guides/kapp-gs/">Getting Started with kapp</a>.</p>
+       </div>   
```

Comment:
`<div class="col">` is missing the closing tag.

```suggestion
   <div class="col">
       <div class="icon">
           <img src="/img/logo.svg" />
       </div>
       <div class="content">
           <p class="strong">Getting Started Guide by Tiffany Jernigan</p>
           <p><a href="https://tanzu.vmware.com/developer/guides/kapp-gs/">Getting Started with kapp</a>.</p>
       </div> 
   </div>
```